README.md

```diff
@@ -425,27 +423,9 @@ an example of how this could look.
 
 For detailed configuration see the [manpage](http://man7.org/linux/man-pages/man5/systemd.slice.5.html)
 
-## Getting help
-
-We encourage you to ask questions on the [mailing list](https://groups.google.com/forum/#!forum/jupyter).
```

Comment:
I'd love for us to leave the discourse forum link in here instead. The badge isn't as discoverable if you don't know what discourse is.

Reply:
Good idea, rebased and updated with ba65b7e